core-services/release-controller/_releases/release-ocp-4.13-ci.json

```diff
@@ -35,9 +35,6 @@
           "ironic-agent",
           "ironic-hardware-inventory-recorder",
           "ironic-inspector",
-          "ironic-ipa-downloader",
-          "ironic-machine-os-downloader",
-          "ironic-static-ip-manager",
```

Comment:
@sherine-k I think we need to keep `ironic-ipa-downloader` here? (it's built from https://github.com/openshift/ironic-ipa-downloader/blob/master/Dockerfile.okd)

And we probably want remove the other two from the FCOS mirroring down below too

Reply:
And I'd say let's not touch 4.10 and 4.11 configs here at all, but let's bring 4.12,4.13 and 4.14 into good shape first

Reply:
`stream8` is the replacement for OCP's ubi/rhel8 here. We could indeed consider switching to `stream9` indeed, since both FCOS and SCOS have new enough kernels. It's not required to do the switch now though

`ironic-hardware-inventory-recorder` has been removed from OCP, but it was still in CI and ended up in OKD releases that way, so we excluded from mirroring (see https://github.com/openshift/release/pull/33696). There's no need to build it anymore.